api/handlers.go

```diff
@@ -1359,13 +1359,22 @@ func handleCollection(db *sql.DB, r ItemRequest) (*Collection, *HttpError) {
 }
 
 func handleLatestContentUnits(db *sql.DB, r BaseRequest) ([]*ContentUnit, []*Collection, *HttpError) {
+	inLangs := []string(nil)
+	for _, lang := range BaseRequestToContentLanguages(r) {
+		inLangs = append(inLangs, fmt.Sprintf("'%s'", lang))
+	}
 	const queryTemplate = `
 WITH CUs AS (
 	SELECT ct.id as type_id, uid, cu_id AS id, film_date
 	FROM ( VALUES (%d), (%d), (%d), (%d), (%d), (%d), (%d) ) ct(id)
 	INNER JOIN LATERAL (
 		SELECT uid, id AS cu_id, coalesce(properties ->> 'film_date', created_at :: TEXT) :: DATE AS film_date
 		FROM content_units cu
+		INNER JOIN (
```

Comment:
is its work for you? 
Look like its condition for inner select and not for select from line 1371

Reply:
Yes it is inner join on content units.

Reply:
I rolled this on staging backend.

Reply:
Other selects are not units (are collections).